system/Debug/Toolbar/Views/toolbar.css

```diff
@@ -186,22 +186,6 @@
   #debug-bar .timeline {
     margin-left: 0;
     width: 100%; }
-    #debug-bar .timeline tr.timeline-parent {
-      cursor: pointer; }
-      #debug-bar .timeline tr.timeline-parent td:first-child nav {
-        background: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAzMCAxNTAiPjxwYXRoIGQ9Ik02IDdoMThsLTkgMTV6bTAgMzBoMThsLTkgMTV6bTAgNDVoMThsLTktMTV6bTAgMzBoMThsLTktMTV6bTAgMTJsMTggMThtLTE4IDBsMTgtMTgiIGZpbGw9IiM1NTUiLz48cGF0aCBkPSJNNiAxMjZsMTggMThtLTE4IDBsMTgtMTgiIHN0cm9rZS13aWR0aD0iMiIgc3Ryb2tlPSIjNTU1Ii8+PC9zdmc+") no-repeat scroll 0 0/15px 75px transparent;
-        background-position: 0 25%;
-        display: inline-block;
-        height: 15px;
-        width: 15px;
-        margin-right: 3px;
-        vertical-align: middle; }
-      #debug-bar .timeline tr.timeline-parent.timeline-parent-open td:first-child nav {
-        background-position: 0 75%; }
-    #debug-bar .timeline tr.timeline-parent.timeline-parent-open {
-      background-color: #DFDFDF; }
-    #debug-bar .timeline tr.child-row:hover {
-      background: transparent; }
```

Comment:
Is the removal of these lines the result of compiling the modified scss files?

Reply:
yes, those were the lines that I manually inserted to the css file in the referenced PR.
they are now in another (hierarchically more suitable) place due to the compiling

Reply:
Thanks!